src/youtube.coffee

```diff
@@ -11,9 +11,12 @@
 module.exports = (robot) ->
   robot.respond /(?:youtube|yt)(?: me)? (.*)/i, (msg) ->
     unless process.env.HUBOT_YOUTUBE_API_KEY
+      robot.logger.error 'HUBOT_YOUTUBE_API_KEY is not set.'
       return msg.send "You must configure the HUBOT_YOUTUBE_API_KEY environment variable"
     query = msg.match[1]
+    robot.logger.debug query
```

Comment:
For debug messages, I like to include some string prefix to explain what it is, ie `youtube query: #{query}`. Without it, you'll just have random strings in the log. Might be worth combining with maxResults below on a single line.